src/client/stylesheets/style.scss

```diff
@@ -383,6 +383,21 @@ a.contact-text:visited {
 	height: 10px;
 }
 
+/* Style inline alias lists the same way as the previously used h4 elements */
+ul.inline-aliases {
+	@extend h4;
+	padding: 0;
+	li {
+		display: inline;
+	}
+	li::after {
+		content: " · ";
+		font-weight: bolder;
+	}
+	li:last-child:after {
+		content: "";
+	}
```

Comment:
I think we might be able to refine the selector to remove the override you currently have for the last child:

`li:not(:last-child)::after`

Reply:
Good point! I also think this should work, but I will test it later nevertheless.